### PR TITLE
Mobile-portfolio-Headline section

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
Add markup for the Mobile portfolio website.
Add stying css file for the Mobile portfolio website.
Add  App-screenshot for the Mobile portfolio website.
Add README File for the Mobile portfolio website.
Add MIT File for the Mobile portfolio website.
Add Image Folder of the Mobile portfolio website

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+  flex-direction: column;
+  background: url(./img/bg1.png) center center/contain no-repeat;
+}
+
+.Portfolio-section1.layout {
+  position: relative;
+  height: -webkit-min-content;
+  height: -moz-min-content;
+  height: min-content;
+}
```
Comment (vechicin) [inline styles.css:154]: I couldn't find the corresponding tag with this class in your HTML file. Kindly remove any unused classes to keep your code clean and organized.

File: portfolio.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            <h1 class="Portfolio-hero-title">I’m Alimony Godfrey Glad to see you!
+            </h1>
+          </div>
+          <div class="Portfolio-highlights2 layout">
+            <h2>
+              I’m a software developer! I can help you build a product , feature
+              or websites Look through some of my work and experience! if you
+              like what you see and have a project you need coded, don't
+              hestiate to contact me.
+            </h2>
```
Comment (vechicin) [inline portfolio.html:57]: According to the evaluation guidelines, this section should be wrapped with a `p` tag.

Screen Shot 2022-05-25 at 8 38 23

Kindly fix this.

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @Godfrey-Alimony  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good PR title/description
- [x] Correct GitHub Flow use
- [x] Good commit history
- [x] Linters checks passing

## Required Changes ♻️

- [ ] Your design, although very similar to the template, is lacking some important details. I think this is due to the fact you're using a lot of `position: relative` to render the layout. I recommend you try to refactor your code so you can position the elements on the page using flexbox **exclusively** as it is a project requirement.

Screen Shot 2022-05-25 at 8 50 11

Here's your design: 
Screen Shot 2022-05-25 at 8 50 59

Here's the template design (I apologize for your colleague's pointers):
Screen Shot 2022-05-25 at 8 52 36


_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by vechicin** [comment]
https://github.com/Godfrey-Alimony/Mobile-Portfolio/blob/a0bbb79f71d8c085a42910aee2cd58acfc492761/README.md?plain=1#L17-L52
[OPTIONAL] I recommend you add instructions on how to get a local copy of this project and how to install the necessary dependencies for it to run properly, it'll make your repo look more professional.

**Comment by vechicin** [comment]
https://github.com/Godfrey-Alimony/Mobile-Portfolio/blob/a0bbb79f71d8c085a42910aee2cd58acfc492761/README.md?plain=1#L54-L59
Be careful with the way you're linking your social media accounts. Kindly fix this as it makes your project look a little unprofessional. The markup for it goes like this: [your_social_media_handle] (link_to_social_media_profile). You need to remove the space between the bracket and the parenthesis for the link to work, I had to add it so you can see how the markup looks like.

File: styles.css (hunk 1 above)
Comment (Godfrey-Alimony) [reply styles.css]: Done

File: portfolio.html (hunk 2 above)
Comment (Godfrey-Alimony) [reply portfolio.html]: Done

**Comment by Godfrey-Alimony** [comment]
> https://github.com/Godfrey-Alimony/Mobile-Portfolio/blob/a0bbb79f71d8c085a42910aee2cd58acfc492761/README.md?plain=1#L54-L59
> 
> 
> Be careful with the way you're linking your social media accounts. Kindly fix this as it makes your project look a little unprofessional. The markup for it goes like this: [your_social_media_handle] (link_to_social_media_profile). You need to remove the space between the bracket and the parenthesis for the link to work, I had to add it so you can see how the markup looks like.

Done

**Comment by Godfrey-Alimony** [comment]
Thank you much sir.

**Comment by Godfrey-Alimony** [comment]
Thank you so much Sir.

**Pull request merged**